plugin/gatsby-node.js

```diff
@@ -1 +1,8 @@
 // https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/
+export const pluginOptionsSchema = ({ Joi }) => {
```

Comment:
**chore:** Needs to be exports

Reply:
bums... ok changed it to `exports.pluginOptionsSchema`